cmd/snap/cmd_snap_op.go

```diff
@@ -201,8 +209,33 @@ func (x *cmdRefresh) Execute([]string) error {
 	return listSnaps([]string{name})
 }
 
+type cmdTry struct {
+	DevMode    bool `long:"devmode" description:"Install in development mode and disable confinement"`
```

Comment:
Is confinement supposed to work with snap try? I've only started thinking about this but it seems if you set INSTALL_DIR appropriately, it should work (owner matches may cause an issue).